partiql-cli/src/main/kotlin/org/partiql/cli/Main.kt

```diff
@@ -237,7 +247,42 @@ internal class MainCommand : Runnable {
             // val connector = LocalPlugin.create(root.toPath())
             // return mapOf("default" to connector)
         }
+
+        if (env != null) {
+            return listOf(parsePartiQL(env!!))
+        }
+
         // Derive a `default catalog from stdin (or file streams)
+        return listOf(parseIon())
+    }
+
+    private fun parsePartiQL(env: File): Catalog {
```

Comment:
nit (naming): `parsePartiQL` and `parseIon` should be called something like `loadPartiQL` or `loadIon`. Parsing generally has a different meaning than what these functions accomplish